### ISSUE TITLE
Issue Summary

### ISSUE THREAD

**Issue by guarguae** [JannisX11/blockbench#1893]
### What are you trying to do, and what do you expect to happen?

I tried to make the json.format modeling automatically load the previously set texture when opened with blockbench software.

### What happens instead?

Conversely, when the model is opened, previously set textures are not loaded correctly.After changing the file or re-importing the texture, saving and opening still does not load the texture correctly.

![PI I7 _$@YPRFTSTBX_9)1N](https://github.com/JannisX11/blockbench/assets/135096243/5388e127-55b0-4ee3-b85e-2dc633daf763)

### Model format in which the issue occurs

json.

### Blockbench variant

Program

### Operating System

win11

### Installed Blockbench plugins

No

**Comment by JannisX11** [comment]
Please attach the resource pack as a zip in order to identify the issue.

**Comment by guarguae** [comment]
The model in this compressed package has the above problem, how to solve it?
[新建压缩(zipped)文件夹.zip](https://github.com/JannisX11/blockbench/files/11609907/zipped.zip)

**Comment by guarguae** [comment]
We found that the texture paths that went wrong were all texture/ (textures we set). This path is different from our initial path. The texture column shows that the texture is not found, you need to add and apply it again, and you still need to repeat the operation process after saving and closing it. It cannot be achieved to open a model where textures have been loaded correctly.

**Comment by guarguae** [comment]
> 请将资源包作为 zip 格式附加，以便确定问题所在。
The model in this archive has the above problems, how to solve it?
https://github.com/JannisX11/blockbench/files/11609907/zipped.zip
We found that the texture paths that went wrong were all textures/(textures we set). This path is different from our original path. The Texture column shows that no texture was found, needs to be added and applied again, and the process still needs to be repeated after saving and closing. Unable to open a model that has textures loaded correctly
